docs/editors/migration.md

```diff
@@ -1,35 +1,39 @@
 # Migrating from `ruff-lsp`
 
-To provide some context, [`ruff-lsp`](https://github.com/astral-sh/ruff-lsp) is the LSP implementation for Ruff to power the editor
-integrations which is written in Python and is a separate package from Ruff itself. The **native
-server** is the LSP implementation which is written in Rust and is available under the `ruff server`
-command. This guide is intended to help users migrate from
-[`ruff-lsp`](https://github.com/astral-sh/ruff-lsp) to the native server.
+To provide some context, [`ruff-lsp`][ruff-lsp] is the [Language Server Protocol] implementation for
```

Comment:
I think we could make this slightly more concise by just giving the context, without the introductory clause:

```suggestion
[`ruff-lsp`][ruff-lsp] is the [Language Server Protocol] implementation for
```